### PR TITLE
Fix closing tags and add rich data for organization, products, aya results list

### PR DESCRIPTION
**Pull request by assem-ch**
(not ready)

### CODE REVIEW

File: src/alfanous-django/templates/base.html [hunk 1]
```diff
@@ -372,8 +372,9 @@ <h1>
                             {% endif %}
                         </form>
                 </div><!--/span4-->
-            </main>
+
             </div><!--/row-->
+            </main>
```
Comment (sneetsher) [inline src/alfanous-django/templates/base.html]: This way the results are not contained in the main block. 

To include results block, We have to include side nav too. only if we take out.

File: src/alfanous-django/templates/base.html [hunk 2]
```diff
@@ -121,6 +121,24 @@
 
     </style>
 
+<script type="application/ld+json">
+{
+  "@context": "http://schema.org",
+  "@type": "Organization",
```
Comment (sneetsher) [inline src/alfanous-django/templates/base.html]: Isn't http://schema.org/SoftwareApplication better to describe Alfanous as name for search engines? Organization is not our mean product or job.

Comment (assem-ch) [reply src/alfanous-django/templates/base.html]: Yes, may be

**Comment by assem-ch** [comment]
I have the 500 error for the itemlist , and i cant test locally, could you review it @sneetsher ?

**Comment by assem-ch** [comment]
@sneetsher error 500 is fixed, and now google can get the rich data:
screen shot 2018-05-26 at 6 05 30 am


Yet the keyboard is not fixed yet

**Pull request merged**